ts/src/bitvavo.ts

```diff
@@ -278,7 +278,7 @@ export default class bitvavo extends Exchange {
         });
     }
 
-    currencyToPrecision (code, fee, networkCode = undefined) {
+    currencyToPrecision (code: string, fee: number, networkCode: Str = undefined): string {
```

Comment:
this is wrong, all toPrecision methods accept a string as the cost/amount/price

Reply:
If that's what you want. Previously we've bee passing numbers to this method. Making the type a string required a lot of code changes, you can look at the last commit if you want to see them all

Reply:
Yeah not so sure anymore, we have cases where we pass numbers and others strings

Reply:
> Yeah not so sure anymore, we have cases where we pass numbers and others strings

so do you think the type should be a number, a string, or both?